Test/Unit/StatusTest.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+namespace Vindi\Payment\Test\Unit;
+
+use \Magento\Sales\Model\Order;
+
+class StatusTest extends \PHPUnit\Framework\TestCase
+{
+
+    protected $objectManager;
+    protected $paymentMock;
+    
+    public function setUp()
+    {
+        $this->objectManager = new \Magento\Framework\TestFramework\Unit\Helper\ObjectManager($this);
+        $this->paymentMock   = $this->createPaymentMock();
+    }
+
+    public function testGetStatusToOrderCompleteWithNoStatusConfigured()
+    {       
+        $this->assertEquals(Order::STATE_PROCESSING, $this->createHelperObjectManager(null)->getStatusToOrderComplete());
```

Comment:
Esse teste deve sofrer uma alteração devido uma falha crítica ajustada no PR #55

Reply:
```suggestion
        $this->assertEquals('pending', $this->createHelperObjectManager(null)->getStatusToOrderComplete());
```

Reply:
Esse teste está sendo feito para o método getStatusToOrderComplete do arquivo Helper/Data.php, que foi o que foi alterado pelo nosso time no último desenvolvimento, então a sugestão não irá funcionar.

Reply:
Acreduti que a sugestão seja para o método testSetProcessingOrderStatusOnPlaceCreditCard(), trocando:

$this->assertEquals(Order::STATE_PROCESSING, $this->paymentMock->getOrder()->getStatus());

Por:

$this->assertEquals('pending', $this->paymentMock->getOrder()->getStatus());

Confirma?

Reply:
Isso faz sentido :smile: